purchase_blanket_order/models/blanket_orders.py

```diff
@@ -269,7 +270,7 @@ def _validate(self):
 
     def set_to_draft(self):
         for order in self:
-            order.write({"state": "draft"})
+            order.write({"state": "draft", "cancelled": False})
```

Comment:
Does not make sense to write the state "draft".

Reply:
@JordiBForgeFlow It's done now